components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -163,10 +212,15 @@ export const PortfolioOverview = () => {
       )
   }, [accounts, filteredOutPortfolioAccountAddresses])
 
+  const accountsListWithRewards =
```

Comment:
memoize arrays to minimize child rerenders

Reply:
Memoized